packages/indexer-client/src/types/serverTypes.ts

```diff
@@ -190,12 +195,20 @@ export interface IndexerServerReferralCodeResponse {
   referral_code: string | null;
 }
 
-export interface IndexerServerFundingRateResponse {
+export interface IndexerServerFundingRate {
   product_id: number;
   funding_rate_x18: string;
   update_time: number;
 }
 
+export type IndexerServerFundingRateResponse = IndexerServerFundingRate;
+
+// Map of productId -> IndexerServerFundingRate
+export type IndexerServerFundingRatesResponse = Record<
```

Comment:
I think it makes sense there since the naming convention seems to be `IndexerServer{EndpointName}Response` but think we should use singular for `IndexerServerFundingRate` since it returns a single value for each product (also so that we can re-use it for both `funding_rate` and `funding_rates`, wdyt?

Reply:
I think that sounds good, will make that change!

Reply:
addressed!